test/v1alpha1/knativekafka.go

```diff
@@ -0,0 +1,106 @@
+package v1alpha1
+
+import (
+	kafkav1alpha1 "github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis/operator/v1alpha1"
+	"github.com/openshift-knative/serverless-operator/test"
+	"github.com/pkg/errors"
```

Comment:
If you have to touch this PR again because of any failures, please remove this dependency. We should be using `fmt.Errorf("blabla: %w, err)` instead of `errors.Wrap`. I have a parallel PR open that fixes other occasions, that's on hold currently.

Reply:
Since this one needs some fixes now, I unheld the other PR, so please make sure to drop this dep.